content/applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst

```diff
@@ -2,62 +2,60 @@
-    :align: center
+.. image:: integrating_landed_costs/landed-costs-setting.png
+   :align: center
+   :alt: Activate the landed cost feature in Inventory Settings
 
 Add costs to products
 =====================
 
 Receive the vendor bill
 -----------------------
 
-Let’s imagine I receive a bill from custom duties for a shipment. I’ll
-tick the box *Landed Costs* on the vendor bill line.
+Let's imagine I receive a bill from custom duties for a shipment. I'll tick the box *Landed Costs*
+on the vendor bill line.
 
-.. image:: integrating_landed_costs/integrating_landed_costs_02.png
-    :align: center
+.. image:: integrating_landed_costs/landed-costs-field-vendor-bill.png
+   :align: center
+   :alt: Enable Landed Costs field on vendor bill line.
 
 .. note::
-         The landed cost product must be of type service.
+   The landed cost product must be of type service.
 
-If this product is always a landed cost, you can also define it on the
-product and avoid having to tick the box on each vendor bill.
+If this product is always a landed cost, you can also define it on the product and avoid having to
+tick the box on each vendor bill.
 
-.. image:: integrating_landed_costs/integrating_landed_costs_03.png
-    :align: center
+.. image:: integrating_landed_costs/define-landed-cost-on-product.png
+   :align: center
+   :alt: Option to define a product as always a landed cost
 
-At the top of my vendor bill, I’ll see a button *create landed costs*.
-I click on this button and a landed cost is automatically created. I can
-now decide on which picking those additional costs should apply.
+At the top of my vendor bill, I'll see a button *create landed costs*. I click on this button and a
+landed cost is automatically created. I can now decide on which picking those additional costs
+should apply.
 
-.. image:: integrating_landed_costs/integrating_landed_costs_04.png
-    :align: center
+.. image:: integrating_landed_costs/warehouse-transfer-landed-costs.png
+   :align: center
+   :alt: Using a warehouse transfer to cover a landed cost in the accounting journal.
 
-I can now click on *Compute* and go in the tab *Valuation
-Adjustments* to see the impact on my products costs. The last step is
-to validate the landed cost.
+I can now click on *Compute* and go in the tab *Valuation Adjustments* to see the impact on my
+products costs. The last step is to validate the landed cost.
 
-I can access the journal entry that has been created by the landed cost
-by clicking on the journal entry.
+I can access the journal entry that has been created by the landed cost by clicking on the journal
+entry.
 
-.. image:: integrating_landed_costs/integrating_landed_costs_05.png
-    :align: center
+.. image:: integrating_landed_costs/landed-cost-journal-entry.png
+   :align: center
+   :alt: Landed cost journal entry
 
 .. note::
    You are not forced to start from the vendor bill, you can also go in :menuselection:`Inventory
```

Comment:
Replace the sentence on Lines 61-62 `You are not forced...create the landed cost from there.` with:

``Landed cost records can also be directly created in :menuselection:`Inventory --> Operations --> Landed Costs`, it is not necessary to create a landed cost record from the vendor bill.``